packages/hash/frontend/src/pages/[account-slug]/[page-slug].page.tsx

```diff
@@ -39,50 +50,143 @@ import { PageIconButton } from "../../components/PageIconButton";
 import { PageLoadingState } from "../../components/PageLoadingState";
 import { CollabPositionProvider } from "../../contexts/CollabPositionContext";
 import {
+  GetAllLatestEntitiesQuery,
   GetPageInfoQuery,
   GetPageInfoQueryVariables,
 } from "../../graphql/apiTypes.gen";
+import { getAllLatestEntitiesQuery } from "../../graphql/queries/knowledge/entity.queries";
+import { apolloClient } from "../../lib/apollo-client";
+import { constructMinimalOrg, MinimalOrg } from "../../lib/org";
+import { constructMinimalUser, MinimalUser } from "../../lib/user";
 import { getLayoutWithSidebar, NextPageWithLayout } from "../../shared/layout";
 import { HEADER_HEIGHT } from "../../shared/layout/layout-with-header/page-header";
 import { useReadonlyMode } from "../../shared/readonly-mode";
-import { useRouteAccountInfo, useRoutePageInfo } from "../../shared/routing";
 import { Button } from "../../shared/ui/button";
 import {
   TOP_CONTEXT_BAR_HEIGHT,
   TopContextBar,
 } from "../shared/top-context-bar";
 
-// Apparently defining this is necessary in order to get server rendered props?
-export const getStaticPaths: GetStaticPaths<{ slug: string }> = () => ({
-  paths: [], // indicates that no page needs be created at build time
-  fallback: "blocking", // indicates the type of fallback
-});
-
 type PageProps = {
+  pageWorkspace: MinimalUser | MinimalOrg;
+  pageEntityId: EntityId;
   blocks: HashBlock[];
 };
 
+type PageParsedUrlQuery = {
+  "account-slug": string;
+  "page-slug": string;
+};
+
+export const tbdIsPageParsedUrlQuery = (
```

Comment:
what does `tbd` mean here? Is this naming intentional?

Reply:
This is a bad habit of mine for writing typeguards, it stands for "to be determined". Have removed it in https://github.com/hashintel/hash/pull/1650/commits/136afbbdd12c5d6af013ab471d1aa06794d80228